docs/test_plan/timed_exam.rst

```diff
@@ -0,0 +1,79 @@
+Timed Exams
+===========
+
+Helpful ReadTheDocs Pages:
+
+- `Configuring a Proctoring Provider <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/proctored_enabling.html#configuring-proctoring-provider>`_
+- `Creating a Proctored Exam <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/pt_create.html#creating-a-proctored-exam>`_
+
+Prerequisite steps:
+    #. Find or create a test course with proctoring enabled and an LTI proctoring provider chosen
+    #. Find or create a test course without an LTI proctoring provider chosen
+    #. Find or create an exam section with a type of 'timed exam' in each course
```

Comment:
As a prerequisite, I would also make sure you have a user enrolled in the verified track for each of these courses